apps/server/src/modules/group/repo/group.repo.ts

```diff
@@ -205,4 +163,39 @@ export class GroupRepo extends BaseDomainObjectRepo<Group, GroupEntity> {
 
 		return domainObjects;
 	}
+
+	private buildPipeline(skip?: number, limit?: number, nameQuery?: string): unknown[] {
```

Comment:
If an empty nameQuery is passed, there's no need to continue building the pipeline...

`  if (!StringValidator.isNotEmptyString(nameQuery)) {
    return [];
  }`

Reply:
the pipeline was built by more than the namequery, now the private function does even more, so i guess this has resolved itself